x-pack/test/accessibility/apps/advanced_settings.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['common', 'settings', 'header']);
+  const a11y = getService('a11y');
+  const testSubjects = getService('testSubjects');
+
+  describe('Stack Management -Advanced Settings', () => {
+    before(async () => {
+      await PageObjects.settings.navigateTo();
+      await a11y.testAppSnapshot();
+    });
```

Comment:
I think it's unusual to put the testAppSnapshot in the before method?  Is this checking the `Stack Management` page and not the `Advanced Settings` page?

Reply:
removed it